src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -137,6 +137,7 @@
       AutomationProperties.AutomationId="Button_Install"
       Click="InstallButton_Clicked"
       Visibility="{Binding InstalledVersion, Converter={StaticResource InverseNullToVisibilityConverter}, Mode=OneWay}"
+      IsEnabled="{Binding IsInstallorUpdateButtonEnabled, Mode=OneWay}"
```

Comment:
``` suggestion
      IsEnabled="{Binding IsSelectedVersionInstalled, Converter={StaticResource InverseBooleanConverter}, Mode=OneWay}"
```
I am a beginner in WPF and have limited knowledge on how bindings work. If I understand this logic correctly, the button is enabled when `IsSelectedVersionInstalled = false`. How about using the existing `InverseBooleanConverter` which returns `true` when `IsSelectedVersionInstalled` returns `false`? If this suggestion is acceptable then we can avoid creating new property. This comment applies to line no. `152`.

Reply:
This looks cleaner to me.  If it works, I'd say use this instead of adding a new property.

Reply:
What I'm trying to fix here is also show the button disabled when the selected version is null, that's why I'm creating a new variable to avoid changing/removing `IsSelectedVersionInstalled`, I'm going to move this code to this [pr](https://github.com/NuGet/NuGet.Client/pull/4355) because with my new feature ([Custom Versions](https://github.com/NuGet/NuGet.Client/pull/4339)) we can have nulls in `SelectedVersion` more often